### PR TITLE
Added version and build timestamp

### PR DESCRIPTION
**Pull request by fernandojsg**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -13,13 +13,31 @@ if (process.env.NODE_ENV === 'dev') {
...
+
 // Minification.
 var plugins = [
   new webpack.DefinePlugin({
     'process.env':{
       'NODE_ENV': JSON.stringify(process.env.NODE_ENV)
-    }
-  })
+    },
+    VERSION: JSON.stringify(require('./package.json').version),
```
Comment (dmarcos) [inline webpack.config.js:37]: do you need `stringify`?

File: webpack.config.js [hunk 2]
```diff
@@ -13,13 +13,31 @@ if (process.env.NODE_ENV === 'dev') {
...
+    BUILD_TIMESTAMP: getBuildTimestamp()
```
Comment (dmarcos) [inline webpack.config.js]: Is it better to use the git hash instead of the timestamp?

**Review by dmarcos — COMMENTED** [review]
It looks good

File: webpack.config.js (hunk 1 above)
Comment (fernandojsg) [reply webpack.config.js:37]: Yeah you're right, not needed, I just copied it from the previous plugin O:)

**Comment by fernandojsg** [comment]
Updated to show hash and a more readable timestamp:
```
A-Frame Inspector Version: 0.4.0 (11-01-2017 Commit: b53002f)
```

**Comment by dmarcos** [comment]
It looks great. We should do the same for a-frame

**Pull request merged**